java_console/io/src/main/java/com/rusefi/SimulatorExecHelper.java

```diff
@@ -99,10 +99,10 @@ public static void startSimulator() {
     }
 
     public static File getSimulatorBinary(String binaryPath) {
-        File binary = new File(binaryPath + SIMULATOR_BINARY_NAME);
+        File binary = new File(binaryPath, SIMULATOR_BINARY_NAME);
 
         if (!binary.exists()) { // try also for Windows/PE executable
-            binary = new File(binaryPath + ".exe");
+            binary = new File(binaryPath, SIMULATOR_BINARY_NAME + ".exe");
```

Comment:
This was the mistake: `binaryPath` is not concatenated to on line 102, so we have to include `SIMULATOR_BINARY_NAME` again on line 105.